From 21aa0c54586f48e3fe51c71138b853977a4d8a66 Mon Sep 17 00:00:00 2001
From: Miroslav Rezanina <mrezanin@redhat.com>
Date: Wed, 6 Nov 2013 12:36:01 +0100
Subject: [PATCH 66/81] monitor: Remove pci_add command for Red Hat Enterprise Linux

RH-Author: Miroslav Rezanina <mrezanin@redhat.com>
Message-id: <2e280a8060b33499eae1704843285ceb7f602470.1383741033.git.mrezanin@redhat.com>
Patchwork-id: 55518
O-Subject: [RHEL7 qemu-kvm PATCH v2 1/4] monitor: Remove pci_add command for Red Hat Enterprise Linux
Bugzilla: 1010858
RH-Acked-by: Michal Novotny <minovotn@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Orit Wasserman <owasserm@redhat.com>
RH-Acked-by: Eric Blake <eblake@redhat.com>

From: Miroslav Rezanina <mrezanin@redhat.com>

This is forwardport of RHEL-6 commit 22aef6c9882ac60bcf3df98e67c4b9b729a90707:

    monitor: Remove pci_add command for Red Hat Enterprise Linux

    pci_add is the old way of hot-plugging pci (nic or storage) devices.
    The newer device_add is the recommended and suggested command.

    Signed-off-by: Amit Shah <amit.shah@redhat.com>

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 hmp-commands.hx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 hmp-commands.hx |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/hmp-commands.hx b/hmp-commands.hx
index 2fc2c0b..d5ec349 100644
--- a/hmp-commands.hx
+++ b/hmp-commands.hx
@@ -1089,7 +1089,7 @@ STEXI
 Add drive to PCI storage controller.
 ETEXI
 
-#if defined(TARGET_I386)
+#if defined(TARGET_I386) && 0 /* Disabled for Red Hat Enterprise Linux */
     {
         .name       = "pci_add",
         .args_type  = "pci_addr:s,type:s,opts:s?",
-- 
1.7.1

